app/assets/stylesheets/style.scss

```diff
@@ -1,2 +1,26 @@
 @import '_govuk-elements';
 
+.search-result {
```

Comment:
are there standard styles we can use from the templates as opposed to overriding font weights and text-decoration ?